packages/clay-pagination/stories/index.tsx

```diff
@@ -53,4 +88,90 @@ storiesOf('ClayPagination', module)
 				totalPages={totalPages}
 			/>
 		);
+	})
+	.add('w/ bar', () => {
```

Comment:
I think it would be best to put `w / bar low-levels` or something of a similar to differentiate with the bass example.